app/tests/test_browse.py

```diff
@@ -568,19 +568,20 @@ def test_browse_consignment_filter_display_multiple_pages(
             "Closure start date",
             "Closure period",
         ],
-        ["-", "test_file6.txt", "closed", "-", "-"],
-        ["-", "test_file7.png", "closed", "-", "-"],
+        ["15/12/2023", "test_file6.txt", "closed", "05/11/2023", "-"],
```

Comment:
whats the equivalent change in the code that led to these dates displaying as non-null?

Reply:
Resolved over call. Flaky test